packages/ngworker/lumberjack/src/lib/log-drivers/lumberjack-log-driver-logger.ts

```diff
@@ -17,9 +19,12 @@ import { LumberjackLogDriverLog } from './lumberjack-log-driver.log';
  */
 @Injectable()
 export class LumberjackLogDriverLogger<TPayload extends LumberjackLogPayload | void = void> {
-  #driverLogStrategyMap = {
-    [LumberjackLevel.Debug]: debugLogDriverLoggingStrategy,
+  /**
+   * A record of logging strategies for each log level.
+   */
+  readonly #loggingStrategies: Readonly<Record<LumberjackLogLevel, LumberjackLogDriverLoggingStrategy>> = {
```

Comment:
It is probably enough to do the following

```suggestion
  readonly #loggingStrategies: Readonly<Record<LumberjackLogLevel, LumberjackLogDriverLoggingStrategy<TPayload>>> = {
```